tests/e2e/smoke-pod-labels/00-install.yaml

```diff
@@ -3,7 +3,7 @@ kind: OpenTelemetryCollector
 metadata:
   name: testlabel
 spec:
-  image: ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-contrib:0.94.0
```

Comment:
Do we want to change this? I thought the point was to use a non-default here? It also won't work because the first k8s distro release will be 0.98.0.